src/components/button/button.ts

```diff
@@ -32,49 +43,55 @@ export class MdButton {
     setTimeout(() => { this.isMouseDown = false; }, 100);
```

Comment:
Also, if the order is focus, then click event, can you listen for the click event and un-focus on click?